### PR TITLE
Update deprecated KeyboardEvent properties

### PR DESCRIPTION
**Pull request by mike-000**
In addition to `KeyboardEvent.charCode` updated by #14492 MDN https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent#obsolete_properties now lists `KeyboardEvent.keyCode` and `KeyboardEvent.which` as deprecated and recommends using `KeyboardEvent.key` instead.

### CODE REVIEW

File: test/browser/spec/ol/interaction/keyboardpan.test.js [hunk 1]
```diff
@@ -31,22 +31,22 @@ describe('ol.interaction.KeyboardPan', function () {
         preventDefault: Event.prototype.preventDefault,
       });
 
-      event.originalEvent.keyCode = 40; // DOWN
+      event.originalEvent.key = 'ArrowDown';
```
Comment (tschaub) [inline test/browser/spec/ol/interaction/keyboardpan.test.js:34]: Can we import and use `Key.DOWN` here?  Same below.

Comment (mike-000) [reply test/browser/spec/ol/interaction/keyboardpan.test.js:34]: Is that normal for tests?  If so should we not also be using `KEYDOWN`, etc. from `ol/events/EventType` in tests?  (A few tests do use `EventType.CHANGE` - in one case inconsistently using both that and the string - but most do not).

Comment (tschaub) [reply test/browser/spec/ol/interaction/keyboardpan.test.js:34]: Good point.  Small screen and I didn’t pay attention to which file this was in.

**Review by tschaub — APPROVED** [review]
Thanks, @mike-000

**Pull request merged**